operate/webapp/src/main/java/io/camunda/operate/webapp/backup/BackupRepository.java

```diff
@@ -19,12 +24,23 @@ public interface BackupRepository {
 
   GetBackupStateResponseDto getBackupState(String repositoryName, Long backupId);
 
-  List<GetBackupStateResponseDto> getBackups(String repositoryName, boolean verbose);
+  List<GetBackupStateResponseDto> getBackups(
+      String repositoryName, boolean verbose, final String pattern);
 
   default List<GetBackupStateResponseDto> getBackups(final String repositoryName) {
-    return getBackups(repositoryName, true);
+    return getBackups(repositoryName, true, null);
   }
 
   void executeSnapshotting(
       BackupService.SnapshotRequest snapshotRequest, Runnable onSuccess, Runnable onFailure);
+
+  static Either<Throwable, String> validPattern(final String pattern) {
+    if (pattern == null || pattern.isEmpty()) {
+      return Either.right("*");
+    } else if (pattern.length() <= 20 && BACKUPID_PATTERN.matcher(pattern).matches()) {
+      return Either.right(pattern);
+    } else {
+      return Either.left(new IllegalArgumentException("Invalid pattern: " + pattern));
```

Comment:
👍🏼 I was already afraid that we just pass it to the ES/OS API and they can do add more query parameters :D